index.js

```diff
@@ -181,3 +185,53 @@ AddonManager.addInstallListener({
     app.send('addon-install:download-failed', formatInstallData(install));
   }
 });
+
+function pingServer(title, addon) {
+  cookieManager.getCookiesFromHost(prefs.HOSTNAME, function(cookies) {
+    let csrf;
+    let cookieStr = '';
+    cookies.forEach(function(c) {
+      cookieStr += c.name + '=' + c.value + ' ';
+      csrf = c.value;
+    });
+
+    getMe(csrf, cookieStr, function(id) {
+      request({
+        url: 'http://' + prefs.HOSTNAME + '/api/metrics/',
+        content: formatEvent(title, id, addon),
+        headers: {
+          'Cookie': cookieStr,
+          'X-CSRFToken': csrf
+        },
+        onComplete: function(response) {
+          console.error(response.text);
+        }
+      }).post();
+    });
+  });
+}
+
+function getMe(csrf, cookieStr, cb) {
+  request({
+    url: 'http://' + prefs.HOSTNAME + '/api/metrics/?format=json',
```

Comment:
We will definitely not want `http` URLs in production, so maybe just add another mostly-redundant pref like prefs.BASE_URL (terrible name, but you get the idea).